test/tests/userobjects/Terminator/tests

```diff
@@ -18,4 +18,31 @@
     issues = '#16452'
     requirement = "The system shall be able to terminate abort the current step in a solve based on post-processing calculations performed within the simulation to cut the timestep."
   [../]
+  [./terminator_message_1]
+    type = 'RunException'
+    allow_warnings = true
+    expect_err = "This is an error!"
+    input = 'terminator_message.i'
+    group = 'requirements'
```

Comment:
Why do we have this? remnants of older SQA?

Reply:
No idea. @aeslaughter ? There are about 33 of those in framework tests.

Reply:
Yes, this is old and can be removed.

Reply:
Ok, I'll add a commit for that